packages/ibm-products/src/components/Datagrid/styles/addons/_RowSizeDropdown.scss

```diff
@@ -45,6 +45,40 @@
 }
 
 .#{variables.$block-class}
-  .#{variables.$block-class}__row-size__row-settings-trigger--open.#{c4p-settings.$carbon-prefix}--btn--ghost {
+  table.#{c4p-settings.$carbon-prefix}--data-table--xl.#{variables.$block-class}__vertical-align-center
+  td,
+.#{variables.$block-class}
+  table.#{c4p-settings.$carbon-prefix}--data-table--xl.#{variables.$block-class}__vertical-align-top
+  td {
+  align-items: flex-start;
+  padding-top: $spacing-05;
+  padding-bottom: $spacing-05;
+
+  &.#{variables.$block-class}__actions-column-cell {
+    padding-left: $spacing-03;
+  }
+}
+
+// Top align cell content for xl row size
```

Comment:
"_Top align **header** content for xl row size_"?

Perhaps include a similar helpful comment for the "cell content" above and the "checkbox row header" below?